cobbler/utils.py

```diff
@@ -1959,7 +1966,7 @@ def dhcpconf_location(api):
         return "/etc/dhcpd.conf"
     elif version[0] in ["fedora"] and version[1] < 11:
         return "/etc/dhcpd.conf"
-    elif dist == "suse":
+    elif dist in ("opensuse tumbleweed", "opensuse leap", "sles"):
```

Comment:
One minor nitpick here. This might break in the future. I think it would be better to use the `make` variable that we are getting in `ver`. If I'm not mistaken, this should be something like `('suse', '15.1')`. And "suse" should always be there, if it is an openSUSE or SLE.

Reply:
I followed action_reposync: if distribution.lower() in ("sles", "opensuse leap", "opensuse tumbleweed")

setup.py now has: "suse" in distro.like()

what is the right way to get the value?
it looks like there are different methods in the code

Reply:
dist is "opensuse leap", so (suse, 15.1) doesn't work without changing api.get_os_details()

Reply:
just committed changes to make os detection cleaner. The code sometimes used api.os_version and sometimes api.get_os_details(). now dist and os_version are used from utils.os_versions(). So hopefully only a single place for capturing os details.